thinc/util.py

```diff
@@ -20,8 +20,16 @@
     import cupy
 
     has_cupy = True
+    cupy_version = Version(cupy.__version__)
+
+    if cupy_version.major >= 10:
+        # fromDlpack was deprecated in v10.0.0.
+        cupy_from_dlpack = cupy.from_dlpack
+    else:
+        cupy_from_dlpack = cupy.fromDlpack
```

Comment:
Alternatively, we could have a module like spaCy's `spacy/compat.py` to deal with this deprecation, I think it's nice to have it in one place.

Reply:
Good idea! I'll move it to a separate module.

Reply:
On second thought, I'll leave that for a fresh PR as it'll touch a bunch of other modules, etc.

Reply:
Done.